chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala

```diff
@@ -48,9 +47,29 @@ class KryoBase(classResolver: ClassResolver, referenceResolver: ReferenceResolve
   protected var strategy: Option[InstantiatorStrategy] = None
 
   val functions: Iterable[Class[_]] =
-    (0 to 22).map { idx =>
-      Class.forName("scala.Function" + idx.toString, true, Thread.currentThread().getContextClassLoader())
-    }
+    List(
+      classOf[Function0[_]],
+      classOf[Function1[_,_]],
+      classOf[Function2[_,_,_]],
+      classOf[Function3[_,_,_,_]],
+      classOf[Function4[_,_,_,_,_]],
+      classOf[Function5[_,_,_,_,_,_]],
+      classOf[Function6[_,_,_,_,_,_,_]],
+      classOf[Function7[_,_,_,_,_,_,_,_]],
+      classOf[Function8[_,_,_,_,_,_,_,_,_]],
+      classOf[Function9[_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function10[_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function11[_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function12[_,_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function13[_,_,_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function14[_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function15[_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function16[_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function17[_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]],
+      classOf[Function18[_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]],
```

Comment:
Just noticed this PR (sorry for poking); Here we jumped `Function19`

Reply:
You have an eagle eye!
Fixed this along with scalafmt